src/applications/personalization/dashboard/containers/MessagingWidget.jsx

```diff
@@ -55,35 +53,28 @@ class MessagingWidget extends React.Component {
     messages = messages.filter(message => message.readReceipt !== 'READ');
 
     const data = messages.map(message => {
-      const id = message.messageId;
-      const rowClass = classNames({
-        'messaging-message-row': true,
-      });
 
       const attachmentIcon = message.attachment ? (
-        <i className="fa fa-paperclip" aria-label="Message has an attachment" />
+        <i
+          className="fa fa-paperclip vads-u-flex--auto"
+          aria-label="Message has an attachment"
+        />
       ) : null;
 
       return {
-        id,
-        rowClass,
-        hasAttachment: attachmentIcon,
-        recipientName: makeMessageLink(message.recipientName, id),
-        senderName: makeMessageLink(message.senderName, id),
-        subject: makeMessageLink(message.subject, id),
-        sentDate: makeMessageLink(formattedDate(message.sentDate), id),
+        senderName: message.senderName,
+        subject: (
+          <div className="vads-u-display--flex">
+            <span className="vads-u-flex--fill">{message.subject}</span>
+            {attachmentIcon}
+          </div>
+        ),
+        sentDate: formattedDate(message.sentDate),
```

Comment:
- `hasAttachment` is no longer needed since we removed the object with that `value` from the `fields` prop.
- `id` and `rowClass` were specific to `<SortableTable>`
- We don't need to call `makeMessageLink` with the new `<Table>` - all it was doing was outputting content in an empty `<a>` tag, which looked weird and was useless without the `href`